tests/test_scheduler_memory_fraction.py

```diff
@@ -50,7 +52,8 @@ def setUpClass(cls):
     @classmethod
     def tearDownClass(self):
         shutil.rmtree("tests/saved-models/")
-        
+
+    @pytest.mark.skip()
```

Comment:
Was this temporary or is there something wrong w/ this test?

Reply:
This is the test that is kinda awkward because it needs to be run alone. I will see if tf2 has any options we didn't have with tf1, possibly something to do with tf.config.get_logical_device_configuration ??